pkg/mcs/resource_manager/server/server.go

```diff
@@ -47,13 +49,17 @@ import (
 )
 
 const (
-	leaderTickInterval = 50 * time.Millisecond
-	tcpNetworkStr      = "tcp"
+	tcpNetworkStr = "tcp"
+	// resourceManagerKeyspaceGroupPrimaryElectionPrefix defines the key prefix for keyspace group primary election.
+	// The entire key is in the format of "/pd/<cluster-id>/microservice/resource_manager/keyspace-group-XXXXX/primary"
+	// in which XXXXX is 5 digits integer with leading zeros. For now we use 0 as the default cluster id.
+	resourceManagerKeyspaceGroupPrimaryElectionPrefix = "/pd/0/microservice/resource_manager/keyspace-group-"
```

Comment:
Will the resource manager have the concept of a keyspace group?

Reply:
Yes, in the assumption of the future, we may separate the resource manager service into several to serve different keyspace groups.